### PR TITLE
Project make people use the client

### PR DESCRIPTION
**Pull request by Jaaslet**
We want the players to atleast try out the client, to reduce server usage. This PR is trying to encourage people to use the client, but not force them to.

- [x] Add client exclusive Pokemon
- [x] Credits
- [x] Wait for Sevii island to be merged
- [x] Added isClientRequirement
- [x] Add way to obtain
- [x] Add NPCs explaining the obtain way is exclusive
- [x] Add link in the browser to the client
- [x] Make sure all links in the game opens in a new tab

### CODE REVIEW

File: src/components/saveSelector.html [hunk 1]
```diff
@@ -14,14 +14,19 @@
                 <img alt="Check out the Wiki!" src="https://img.shields.io/badge/wiki-link-blue?logo=wikimedia-commons&style=for-the-badge">
             </a>
         </div>
+        <div id="use-our-client-message" style="color: white; display: none;">
+            To help reduce serverload, please try our client using by <a href="https://github.com/RedSparr0w/Pokeclicker-desktop/releases/latest" target="_blank">clicking here</a>.
+            You can event find an client exclusive pokémon when you reach Sevii Island!
```
Comment (RedSparr0w) [inline src/components/saveSelector.html:19]:
```suggestion
            To help reduce server load, please download our desktop client <a href="https://github.com/RedSparr0w/Pokeclicker-desktop/releases/latest" target="_blank">here</a>.
            You can event find a Pokémon exclusive to the client when you reach the Sevii Islands!
```

**Comment by erzads** [comment]
I didn't know there was a client. Where can I find it?

**Comment by Aegyo** [comment]
https://github.com/RedSparr0w/Pokeclicker-desktop/releases/latest

**Review by Aegyo — CHANGES_REQUESTED** [review]
Missing credit for the ~~pokemon and~~ island image~~s~~ (unless I'm blind?)

**Comment by Jaaslet** [comment]
Credits added and merge conflicts resolved

**Review by Aegyo — APPROVED** [review]

**Pull request merged**